integration-tests/.depcheckrc

```diff
@@ -0,0 +1,34 @@
+ignores: [
```

Comment:
I'll make a separate issue for reviewing these dependencies and making sure that they all belong in this ignore list.